docs/dev/perl5/news/2013/perl-5.16.3.html

```diff
@@ -0,0 +1,70 @@
+[% page.title = "Perl 5.16.3 Release Announcement" %]
+<meta charset="utf-8">
+
+<p>
+We are happy to announce perl 5.16.3 the 4th release of version 16 of Perl
+5.
+</p>
+
+<p>
+You will soon be able to download Perl 5.16.3 from your favorite CPAN
+mirror or find it at:
+</p>
+
+<p>
+https://metacpan.org/release/RJBS/perl-5.16.3/
+</p>
+
+<p>
+SHA1 digests for this release are:
+</p>
+
+<p>
+<pre>
+    83678adf56d3dc51f47a90444a891f4fe16868da  perl-5.16.3.tar.gz
+    060bc17cf9f142d043f9bf7b861422ec624875ea  perl-5.16.3.tar.bz2
+</pre>
+</p>
+
+<p>
+You can find a full list of changes in the file "perldelta.pod" located in
+the "pod" directory inside the release and on the web.
+</p>
+
+<p>
+Perl 5.16.3 represents approximately 4 months of development since Perl
+5.16.2 and contains approximately 870 lines of changes across 39 files from
+7 authors.
+</p>
+
+<p>
+Perl continues to flourish into its third decade thanks to a vibrant
+community of users and developers. The following people are known to have
+contributed the improvements that became Perl 5.16.3:
+</p>
+
+<p>
+Andy Dougherty, Chris 'BinGOs' Williams, Dave Rolsky, David Mitchell,
+Michael Schroeder, Ricardo Signes, Yves Orton.
+</p>
+
+<p>
+The list above is almost certainly incomplete as it is automatically
+generated from version control history. In particular, it does not include
+the names of the (very much appreciated) contributors who reported issues
+to the Perl bug tracker.
+</p>
+
+<p>
+For a more complete list of all of Perl's historical contributors, please
+see the AUTHORS file in the Perl source distribution.
+</p>
+
+<p>
+The next major stable release of Perl 5, version 18.0, should appear in May
+2013.
+</p>
+
+<p>
+rjbs
```

Comment:
It's a little weird how `rjbs` is just down here at the bottom.

**I** know it's his email signature, but it needs some sort of delimiter or explanation or styling.  See for example this one: https://dev.perl.org/perl5/news/2004/perl-5.8.3.html